### ISSUE TITLE
something trying to import 'fs' breaking react-native support

### ISSUE THREAD

**Issue by mnzaki** [jolocom/jolocom-sdk#15]
### Describe the bug
```
error: Error: Unable to resolve module `fs` from `node_modules/@jolocom/sdk/js/src/lib/storage/index.js`: fs could not be found within the project.
A clear and concise description of what the bug is.
```
### To Reproduce
import the SDK in a react-native project

### Side notes
need to have a react-native test case as part of integration tests

**Comment by clauxx** [comment]
[Location](https://github.com/jolocom/jolocom-sdk/blob/98a0fa0feed7954a41d7592a1a26efa3fe13fe31/src/lib/storage/index.ts#L94) where it's used

**Comment by chunningham** [comment]
closed by #18
